db/organizations/appnexus.eno

```diff
@@ -1,11 +1,12 @@
-name: AppNexus Inc.
-website_url: https://www.appnexus.com/en
-privacy_policy_url: https://www.appnexus.com/en/company/platform-privacy-policy
+name: Microsoft
```

Comment:
Since there is already an entry for Microsoft, I would propose to leave this as is (only keep the notes). 

We can mark it as "archived" like that:

```
--- notes
AppNexus acquired by Xandr (AT&T, 2018), acquired by Microsoft (2021). https://en.wikipedia.org/wiki/AppNexus
--- notes
--- notes

archived
```

The existing AppNexus pattern could instead link to organization/microsoft.eno. I created a separate issue for it: https://github.com/ghostery/trackerdb/issues/174 (directly changing it now could break tests; but I'll look into it)